solar-land/scss/layout/_header.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../../shared/scss/components/button";
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1rem 2rem;
+  background-color: #fff;
+
+  .logo {
+    img {
+      height: 40px;
+      width: auto;
+    }
+  }
+
+  .navigation {
+    ul {
+      display: flex;
+      list-style: none;
+      gap: 2rem;
```

Comment:
For this one too it would be better to use $spases list

Reply:
Ok bro :)